delta_othello/game_mechanics.py

```diff
@@ -260,13 +244,22 @@ def __init__(
         ] = choose_move_randomly,
         verbose: bool = False,
         render: bool = False,
+        game_speed_multiplier: float = 1.0,
     ):
         self._board_visualizer = np.vectorize(lambda x: "X" if x == 1 else "O" if x == -1 else "*")
         self._opponent_choose_move = opponent_choose_move
-        self._screen = None
         self.verbose = verbose
         self.render = render
-        self.reset()
+        self.game_speed_multiplier = game_speed_multiplier
+        if self.render:
+            self.init_graphics()
+
+    def init_graphics(self) -> None:
+        pygame.init()
+        self.screen = pygame.display.set_mode((N_ROWS * SQUARE_SIZE, N_COLS * SQUARE_SIZE))
+        pygame.display.set_caption("Othello")
+        self.clock = pygame.time.Clock()
```

Comment:
we could, seems like an unnecessary faff that would make this inconsistent with other games. I proposed just deleting it

Reply:
Eh fine by me, I think using it _might_ fix a minor issue where my `pygame` thinks the game has crashed because nothing is running. Would have to test this myself first, so will ticket it